### PR TITLE
 #167515607  Change Unit Testing to Use Mocha and Chai

### PR DESCRIPTION
**Pull request by Adekoreday**
#### What does this PR do?
This PR implements unit test using mocha and chai.
This chore is raised to enable serving the swagger API-docs via the API URL which was problematic in jest

#### Description of Task proposed in this pull request?

The unit test which was performed using Jest was replaced with mocha-chai

#### How should this be manually tested (Quality Assurance)?
```
npm test
```

#### What are the relevant pivotal tracker stories?
- [167515607](https://www.pivotaltracker.com/story/show/167515607)

#### Any background context you want to add (Operations Impact)?

 `npm install`

#### What I have learned working on this feature:
- I learnt how to set up the unit test using `mocha` and `chai`

### CODE REVIEW

**Review by chialuka — APPROVED** [review]
Good job Korede

**Review by abdulfatai360 — APPROVED** [review]
Your code looks good to me. Well done

File: package.json [hunk 1]
```diff
@@ -1,29 +1,19 @@
 {
-  "name": "Rambo-AuthorsHaven",
+  "name": "rambo-authors-haven",
```
Comment (dinobi) [inline package.json:2]: why this change?

**Review by dinobi — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (Adekoreday) [reply package.json:2]: eslint was flagging it as an error will send you the screenshot
